src/Paynow/Exception/Error.php

```diff
@@ -7,25 +7,25 @@ class Error
     /**
      * @var string
      */
-    private $errorType;
+    private $type;
 
     /**
      * @var string
      */
     private $message;
 
-    public function __construct($errorType, $message)
+    public function __construct($type, $message)
     {
-        $this->errorType = $errorType;
+        $this->type = $type;
         $this->message = $message;
     }
 
     /**
      * @return string
      */
-    public function getErrorType()
+    public function getType()
```

Comment:
since this is a part of public API and also introduces BC break and if this lib follows semantic versioning, then the next proposed version should be bumped to next major version, not major which follows BC

Reply:
I think it's not a BC because the method was never used before.

Reply:
Since this is public method and part of API of this lib, you cannot have any knowledge if someone use it or not. You change something that lib user might be using -> you introduce BC.

Reply:
Ok, I described that change on log

Reply:
@emilleszczak2 I agree with @amodliszewski that's a part of public API If you don't wanna bump major then you should annotate an old method for deprecation and plan it's removal while in the meantime introduce a new method pointer out in old ones deprecation message or smth like that, and only then you're free to bump minor without consequences.

Reply:
I'm gonna release major version that's why I removed the method. I will do that in separate PR.